server/api.py

```diff
@@ -93,23 +94,25 @@ def read_map(claims, id):
 
         Returns a HTTP response.
         """
-        email, user, map = None, None, None
+        email, user, game_map = None, None, None
         try:
             email = claims["email"]
             user = User.objects(email=email).first()
         except Exception as e:
             return malformed_request()
 
         try:
-            map = Map.objects.get(id=id, user=user)
+            game_map = GameMap.objects(id=id).first()
         except (StopIteration, DoesNotExist) as e:
+            current_app.logger.error(e)
```

Comment:
you're gonna probably want to add 
if not current_app.testing:
to all these lines

Reply:
I don't understand why we should do that. If a error occurs we would want to see it in the logs. If something broke we'd lose why it happened.

Reply:
it's only while you're testing that it doesn't print anything out. when it's not testing the log still appends to the file. we would see the error anyways running the tests because the assertion will fail

Reply:
OK. That makes sense.